frontend/src/metabase/pulse/components/RecipientPicker.jsx

```diff
@@ -82,15 +86,9 @@ export default class RecipientPicker extends Component {
                 <span className="ml1">{option.value.common_name}</span>
               </div>
             )}
-            filterOption={(option, filterString) =>
-              // case insensitive search of name or email
-              ~option.value.common_name
-                .toLowerCase()
-                .indexOf(filterString.toLowerCase()) ||
-              ~option.value.email
-                .toLowerCase()
-                .indexOf(filterString.toLowerCase())
-            }
+            // Just a note, but I think logic that filter options by either common_name or email might already work.
+            // But making this close to what we have in UserPicker would be better because that one is more readable.
+            filterOption={filterOption}
```

Comment:
Refactor this part to make it similar to another place and improve redability.

https://github.com/metabase/metabase/blob/6da7a41500daf73943b29797f486f9f91658ffa3/frontend/src/metabase/components/UserPicker/UserPicker.jsx#L51-L56